Server/ConcordiaCurriculumManager/Models/Curriculum/Course.cs

```diff
@@ -63,7 +63,7 @@ public static Course CloneCourseForDeletionRequest(Course course)
             CourseNotes = course.CourseNotes,
             CourseState = CourseStateEnum.CourseDeletionProposal,
             Version = course.Version + 1,
-            Published = course.Published,
+            Published = false,
```

Comment:
I think this change is actually wrong, because when we display the changes, we'll want to see that we've deleted a published course, meaning this should stay as `course.Published`.

Reply:
I thought Published meant that this course is officially published publicly in Concordia's Calendar. I'll change it back.

Reply:
Yes you're completely right, but that's information we want to preserve. "This course that has been deleted was a published course."